cms/templates/index.html

```diff
@@ -443,10 +443,25 @@ <h3 class="course-title">${library_info['display_name']}</h3>
       <div class="notice notice-incontext notice-instruction notice-instruction-nocourses list-notices libraries-tab">
         <div class="notice-item">
           <div class="msg">
-            <div class="copy">
-              <p>${_("You don't have any content libraries yet.")}</p>
-            </div>
+              <h3 class="title">${_("Expecting to see a library already here?")}</h3>
+              <div class="copy">
+                  <p>${_('You will need to be added to the library in {studio_name} by the library creator. Please get in touch with the library creator or administrator for the specific library you are helping to author.').format(studio_name=settings.STUDIO_SHORT_NAME)}</p>
+              </div>
+          </div>
+        </div>
+        <div class="notice-item has-actions">
+          <div class="msg">
+              <h3 class="title">${_('Create Your First Library')}</h3>
+              <div class="copy">
+                  <p>${_('Libraries hold problems and content that can be re-used across multiple courses. Create your first library with the click of a button!')}</p>
```

Comment:
"Libraries hold a pool of components that can be re-used across multiple courses".
(Suggested wording is slightly more in line with existing documentation that talks only about "components" in libraries".) @marcotuts - Is explicit mention of "re-use across multiple courses" OK?